index.html

```diff
@@ -44,129 +65,150 @@ <h4 class="date">Published on Feb 27, 2014</h4>
+                    329,285 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-08.jpg">
-                <h3>Top 10 Linux Job Interview Questions</h3>
-                <small>Computerphile</small>
-                <small>2,129,277 views</small>
+                <p><strong>Top 10 Linux Job Interview Question</strong><br>
+                    <span class="text-font"><small>Computerphile<br>
+                        2,129,277 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-09.jpg">
-                <h3>Running an SQL Injection Attack - Computerphile</h3>
-                <small>Computererphile</small>
-                <small>2,217,477 views</small>
+                <p><strong>Running an SQL Injection Attack - Computerphile</strong><br>
+                    <span class="text-font"><small>Computererphile<br>
+                    2,029,985 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-01.jpg">
-                <h3>Contributing to Open Source Part II: The Real Way</h3>
-                <small>The Odin Project</small>
-                <small>29,985 views</small>
+                <p><strong>Contributing to Open Source Part II: The Real Way</strong><br>
+                    <span class="text-font"><small>The Odin Project<br>
+                    29,985 views </small></span>
+                <p>
+                
             </aside>
             
             <aside>
                 <img src="images/side-image-02.jpg">
-                <h3>'How to Get a Job at the Big 4 - Amazon, Facebook, Google & Microsoft' by</h3>
-                <small>imtiana</small>
-                <small>929,985 views</small>
+                <p><strong>'How to Get a Job at the Big 4 - Amazon, Facebook, Google & Microsoft' by</strong><br>
+                    <span class="text-font"><small>imtiana<br>
+                    929,985 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-03.jpg">
-                <h3>Programming in Visual Basic .Net How to Connect Access Database to</h3>
-                <small>iBasskung</small>
-                <small>16,129,985 views</small>
+                <p><strong>Programming in Visual Basic .Net How to Connect Access Database to</strong><br>
+                    <span class="text-font"><small>iBasskung<br>
+                    16,129,985 views</small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-04.jpg">
-                <h3>REST API concepts and examplesy</h3>
-                <small>WebConcepts</small>
-                <small>3,529,985 views</small>
+                <p><strong>REST API concepts and examplesy</strong><br>
```

Comment:
Don't use the <strong> tag to make regular text bold, use CSS instead. [The HTML Strong Importance Element (<strong>) indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)